.gitignore

```diff
@@ -1,3 +1,4 @@
 .Rproj.user
 .Rhistory
 .RData
+.DS_Store
```

Comment:
Fine to add this here. But it is platform-specific so should be in your global .gitignore at `~/.gitignore`.